Dockerfile

```diff
@@ -0,0 +1,75 @@
+################################################################################
+# Base image
+# Based on https://github.com/andrewmclagan/nginx-hhvm-docker
+################################################################################
+
+FROM nginx
+
+################################################################################
+# Add HHVM repo
+################################################################################
+
+RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
+RUN apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0x5a16e7281be7a449 && \
+    echo deb http://dl.hhvm.com/debian jessie main | tee /etc/apt/sources.list.d/hhvm.list
+
+################################################################################
+# Install supervisor, HHVM & tools
+################################################################################
+
+RUN apt-get install debian-archive-keyring
+RUN apt-get update -o Retries=25
+RUN apt-get install -my -o Retries=25 \
+	hhvm \
```

Comment:
Do we gain much by using HHVM? I'm not opposed to using it but I don't think most of the team has experience with it. (I know I don't!). I personally would be more comfortable using PHP 7.